lib/ansible/modules/storage/netapp/na_ontap_cluster_peer.py

```diff
@@ -114,7 +114,7 @@ def __init__(self):
 
         self.module = AnsibleModule(
             argument_spec=self.argument_spec,
-            required_together=['source_intercluster_lif', 'dest_intercluster_lif', 'passphrase'],
+            required_together=(['source_intercluster_lif', 'dest_intercluster_lif', 'passphrase']),
```

Comment:
According to documentation, we should have two sets of [ ] :

required_together
This is self explanatory, all specified parameters need to be specified together:

required_together=[
  [ 'a', 'b' ]
]